src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+body {
+  background: #e5e5e5;
+  margin: 0;
+  min-width: 375px;
+  max-width: 1200px;
+}
+
+.header {
+  box-sizing: border-box;
+  justify-content: space-between;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.logo {
+  font-size: 0;
+  padding-top: 6px;
+  padding-bottom: 6px;
+}
+
+.nav {
+  position: relative;
+  margin: 0;
+}
+
+.nav__list {
+  display: flex;
+  cursor: pointer;
+  text-transform: uppercase;
+  margin: 0;
+}
+
+.nav__item {
+  list-style: none;
+  text-decoration: none;
+  text-align: center;
+  margin-left: 20px;
+}
+
+.nav__link {
+  color: #000;
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+}
+
+.nav__link::after {
+  display: block;
+  content: "";
+  background: #00acdc;
+  border-radius: 8px;
+  transition: 300ms;
+  height: 4px;
+  position: absolute;
+  bottom: 0;
+}
+
+.nav__link:hover::after {
+  width: 37px;
+  color: #00acdc;
+}
+
+.is-active::after {
+  width: 37px;
```

Comment:
more correctly set width 100% so that it stretches to the width of the element
width 37px
![image](https://user-images.githubusercontent.com/54936044/228375684-66f52eec-06f3-4945-af9a-93a5eaac168f.png)
width 100%
![image](https://user-images.githubusercontent.com/54936044/228376126-e8f83945-3876-4c17-ab4c-f6f7d3497d4f.png)